urbanbroccoli/sor.py

```diff
@@ -31,13 +31,13 @@ def sor_1d(rho, hx, epsilon, maxiter, maxerr, w):
     return phi
 
 def sor_2d(rho, hx, hy, epsilon, maxiter, maxerr, w):
-    """Performs a sucsesive over relaxation for the poisson equation of a two dimensional discretized charge distribution:
+    """Performs a sucsessive over-relaxation for the Poisson equation of a two dimensional discretized charge distribution:
```

Comment:
This line has the correct indention but is too long. Use a very short description and, if necessary, a longer text block after two line breaks.